audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractTest.java

```diff
@@ -104,256 +112,245 @@
 @Import(KafkaContractTest.KafkaInitializer.Config.class)
 @ContextConfiguration(initializers = {KafkaContractTest.KafkaInitializer.class})
 @ActiveProfiles({"akka-test"})
+//@ExtendWith(MockitoExtension.class)
 class KafkaContractTest {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaContractTest.class);
-    private static final ObjectMapper MAPPER = new ObjectMapper();
 
-    // Serialiser for upstream test input
-    static class RequestSerializer implements Serializer<JsonNode> {
-        @Override
-        public byte[] serialize(final String s, final JsonNode auditRequest) {
-            try {
-                return MAPPER.writeValueAsBytes(auditRequest);
-            } catch (JsonProcessingException e) {
-                throw new SerializationFailedException("Failed to serialize " + auditRequest.toString(), e);
-            }
-        }
-    }
-
-    @Autowired
-    private TestRestTemplate restTemplate;
+    @Autowired()
+    Serializer<String> keySerializer;
     @Autowired
-    private ActorSystem akkaActorSystem;
+    Serializer<JsonNode> valueSerializer;
```

Comment:
Got rid of brackets. 
Does the test fail for you? The beans are configured in the Config class at the bottom of the test. If they were not injected, the tests would fail.